elodie/filesystem.py

```diff
@@ -342,31 +374,10 @@ def process_file(self, _file, destination, media, **kwargs):
         file_name = self.get_file_name(media)
         dest_path = os.path.join(dest_directory, file_name)
 
-        db = Db()
-        checksum = db.checksum(_file)
-        if(checksum is None):
-            log.info('Could not get checksum for %s. Skipping...' % _file)
+        new_checksum = self.process_checksum(_file, allow_duplicate)
+        if(new_checksum is None):
```

Comment:
I would add a log message here `log.info('New checksum returned None for %s. Skipping...', _file)`.